### ISSUE TITLE
Feat: Validation error during training

### ISSUE THREAD

**Issue by davinci26** [davinci26/Pytorch-Playground#1]
**Scenario**

The user wants to log and view on console output both training and validation error during training. With this feature the user can know if the model is overfitting or if the model is under trained.

**Expected output**

During training the following log is available
`[{Epoch}, {Batch}] Training loss: {loss_train} Validation loss: {loss_validation}`
